packages/content/src/requests/items/deleteAttribute/DeleteAttributeRequestItem.ts

```diff
@@ -0,0 +1,28 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { CoreId, ICoreId } from "@nmshd/transport";
+import { RequestItemJSON } from "../..";
+import { IRequestItem, RequestItem } from "../../RequestItem";
+
+export interface DeleteAttributeRequestItemJSON extends RequestItemJSON {
+    "@type": "DeleteAttributeRequestItem";
+    attributeId: string;
+}
+
+export interface IDeleteAttributeRequestItem extends IRequestItem {
+    attributeId: ICoreId;
+}
+
+@type("DeleteAttributeRequestItem")
+export class DeleteAttributeRequestItem extends RequestItem implements IDeleteAttributeRequestItem {
+    @serialize()
+    @validate()
+    public attributeId: CoreId;
+
+    public static from(value: IDeleteAttributeRequestItem | Omit<DeleteAttributeRequestItemJSON, "@type">): DeleteAttributeRequestItem {
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here